src/cli/check.rs

```diff
@@ -74,8 +75,12 @@ pub enum FileType {
     /// Auto.
     #[default]
     Auto,
+    /// Raw text.
+    Raw,
     /// Markdown.
     Markdown,
+    /// HTML.
+    Html,
     /// Typst.
     Typst,
 }
```

Comment:
This doesn't have to be fixed / changed in this feature, but what do you think of the following:

+ add a `Text` variant that matches `.txt` files;
+ ignore `Raw` file types by defaults (because some tools would probably call `ltrs` will many files as input, like `pre-commit`, and it doesn't make sense to check for errors in files like `.zip`, code files, or else);
+ add CLI options `--include-raw` and `--include=ext` where the latter includes files with `ext` extension, using `Raw` logic.

?

Reply:
Sounds alright yeah. Maybe it shouldn't be "raw" then, more just "unknown" or "unsupported", what do you think? I was thinking "raw" just because it would be sent as-is, but if there's a separate "text" option I think that would be more clear

Reply:
Yes, I think `Unknown` sounds perfect! I agree that `Raw` conveys the message greatly, my main point was about not checking arbitrary file types, at least by default.

Reply:
All good. I can do this in a separate PR after this one then.